x-pack/legacy/plugins/alerting/server/routes/update.ts

```diff
@@ -32,6 +33,14 @@ export function updateAlertRoute(server: Hapi.Server) {
         },
         payload: Joi.object()
           .keys({
+            throttle: Joi.alternatives()
+              .try(
+                Joi.string().regex(SECONDS_REGEX, 'seconds'),
```

Comment:
hmmm ... can we share this same code in `create()` via a library function, somehow?